x-pack/plugins/apm/public/hooks/use_transaction_latency_chart_fetcher.ts

```diff
@@ -42,14 +58,15 @@ export function useTransactionLatencyChartsFetcher() {
               end,
               transactionType,
               transactionName,
-              uiFilters: JSON.stringify(uiFilters),
+              uiFilters: JSON.stringify(modifiedUiFilters),
```

Comment:
Can't we make `environment` a new property for this API? This would start the work to remove the uiFilters.